### DISCUSSION TITLE
Add fields or render fields required from the value of another field

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#2890] [Help] [answered]
Could you give me an example of code with the reactive method to display a field from the value of another field in a form
Same question on field becaming required from the value from another field

**Comment by eaziweb95** [comment, marked as answer]
is there any answer ?

**Reply by danharrin** [reply]
`Placeholder::make('test')->content(fn ($get) => $get('another_field'))`

`TextInput::make('test')->required(fn ($get) => $get('another_field'))`

Your questions are vague, so it's best just to [read the docs](https://filamentphp.com/docs/2.x/forms/advanced#using-closure-customisation).

**Reply by eaziweb95** [reply]
Thanks a lot, i will try these examples

**Reply by eaziweb95** [reply]
It works for require
 Forms\Components\Radio::make('code')->label(__('Code de la route'))->required(fn ($get) => $get('permis') == 'Non')

But for the Placeholder, how can i put the RadioField to display it when the another_field as a Value

This code doesn't work
Placeholder::make('test')->content(fn ($get) => $get('type_demande_id') == 4)
    Forms\Components\Radio::make('creation_entreprise')->label(__('Création d\'entreprise'))->required(fn ($get) => $get('type_demande_id') == 4)
     ->options(['Oui' => 'Oui', 'Non' => 'Non'])->inline()
   ),

**Reply by eaziweb95** [reply]
Issue is resolved with hidden method, you can close this discussion

                    Forms\Components\Radio::make('creation_entreprise')->label(__('Création d\'entreprise'))
                        ->required(fn ($get) => $get('type_demande_id') == 4)
                        ->hidden(fn (Closure $get) => $get('type_demande_id') != 4)
